Tests/DataParser/test_dataParser.py

```diff
@@ -0,0 +1,116 @@
+import os
+import json
+import xmltodict
+import zipfile
+import pandas as pd
+from pathlib import Path
+import pytest
+import sys
+sys.path.append("../../DataParser")
```

Comment:
Don't add these in between import statements (you can add after all imports are done).  Also, do not add paths like ../../ since it would not be scalable when used by others. Add something where it would not matter where the user runs from.